components/core/tools/scripts/lib_install/check-cmake.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Get the installed cmake version string
+cmake_version=$(cmake --version | head -n1 | awk '{print $3}')
```

Comment:
```suggestion
cmake_version=$(cmake --version | head --lines 1 | awk '{print $3}')
```

To stay consistent with our convention of using long-form options in scripts. This option is also available on macOS: https://ss64.com/mac/head.html